common/app/controllers/EmailSignupController.scala

```diff
@@ -79,6 +100,43 @@ class EmailFormService(wsClient: WSClient, emailEmbedAgent: NewsletterSignupAgen
     }
   }
 
+  def submitWithMany(form: EmailFormManyNewsletters)(implicit request: Request[AnyContent]): Future[WSResponse] = {
```

Comment:
👍  I've done some refactoring - here and in the EmailSignupController. 
It might be possible to reduce the duplication further if the two variants of the form had a common type, but not sure if that would be making things more complex?